requirements.txt

```diff
@@ -30,4 +30,5 @@ packaging>=20.4
 # not directly required, pinned by Snyk to avoid a vulnerability
 wheel>=0.38.0
 # not directly required, pinned by Snyk to avoid a vulnerability
-pygments>=2.7.4 
+pygments>=2.7.4
+albumentations>=1.3.0
```

Comment:
Do we want to add every augmentation plugin as a hard requirement? 
Maybe try to import albumentations in we encounter "Albumentations" in the transforms, and fail nicely if albumentations is missing? E.g we initialize the Albumentations factory on first call.

Reply:
sure, modified in 23a5d578de548870d10e193d7c5bd79e8cbf82c5